### PR TITLE
Updated abp.io document for commercial abp.io

### PR DESCRIPTION
**Pull request by AkinSabriCam**

### CODE REVIEW

File: framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppTemplateBase.cs [hunk 1]
```diff
@@ -10,7 +10,11 @@ public abstract class AppTemplateBase : TemplateInfo
         protected AppTemplateBase(string templateName)
             : base(templateName, DatabaseProvider.EntityFrameworkCore, UiFramework.Mvc)
         {
-            DocumentUrl = "https://docs.abp.io/en/abp/latest/Startup-Templates/Application";
+            if(templateName=="app-pro")
+                DocumentUrl= "https://docs.abp.io/en//commercial/latest";
+            else
+                DocumentUrl = "https://docs.abp.io/en/abp/latest/Startup-Templates/Application";
+
```
Comment (maliming) [inline framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppTemplateBase.cs]: hi 

I think we should modify it in a derived class of `Template`. Like `ModuleProTemplate`.

https://github.com/abpframework/abp/blob/f901f909192aed7fad7d7635ebd8de3b3c6de26c/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/Module/ModuleProTemplate.cs#L16

Comment (AkinSabriCam) [reply framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppTemplateBase.cs]: Hi thank you for your feedback.
If I modify it in ModuleProTemplate , I would change just pro doc link of moduleTemplate thats why I changed it in AppTemplateBase
am i wrong? waiting  for your feedback

Comment (maliming) [reply framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppTemplateBase.cs]: hi @AkinSabriCam 
I mean setting `DocumentUrl` in the constructor of the `AppProTemplate `class.

Comment (AkinSabriCam) [reply framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppTemplateBase.cs]: Hi @maliming 
I got it now. 
I will modify it.  Thank you

**Pull request merged**